.github/requirements-ci-molecule.txt

```diff
@@ -0,0 +1,2 @@
+molecule>=6.0
```

Comment:
please as discussed on the PR review call let's add a comment stating we are waiting for pip 25.1 to move this to a dependency group